.github/workflows/encryption-tests.yml

```diff
@@ -0,0 +1,50 @@
+name: Encryption Tests 
+
+on:
+  push
+  #workflow_dispatch: {}
```

Comment:
Intentionally commented out?

Reply:
Just so any push triggers the action, I've added this branch name into the push config for clarity  - will remove it when we're ready to merge.

Reply:
Do we want to be able to trigger the action manually, or is on "push" / "pr to master" sufficient?

Reply:
I think manually would be nice especially as we continue encryption testing development and if Val or another mongoose contributor adds any encryption functionality.